modelservice/src/main/sql/build-db.sql

```diff
@@ -58,4 +58,18 @@ create table user (
   password     varchar(100) not null,
   role         varchar(100),
   enabled      boolean
-);
\ No newline at end of file
+);
+
+create table metric_tags (
+tag_id        int NOT NULL AUTO_INCREMENT PRIMARY KEY , 
+tag_keys      varchar(255) NOT NULL, 
+tag_values    varchar(255) NOT NULL,
+unique index (tag_keys, tag_values)
+);
+
+create table metric_mapper(
+map_id        int NOT NULL auto_increment PRIMARY KEY,
+id            int NOT NULL,
+tag_id        int NOT NULL,
```

Comment:
use foreign key here. checkout the other mapping table for example

Reply:
Done